ide/static/ide/js/ib/utils.js

```diff
@@ -31,11 +31,26 @@
         };
     };
 
-    IB.ColourWhite = new IB.Colour('GColorWhite', 'white', gettext('White'));
-    IB.ColourBlack = new IB.Colour('GColorBlack', 'black', gettext('Black'));
-    IB.ColourClear = new IB.Colour('GColorClear', 'rgba(0, 0, 0, 0)', gettext('Transparent'));
+    IB.ColourModes = {
+        Colour: 0,
+        Monochrome: 1
+    };
+    IB.ColourMode = null;
```

Comment:
I agree, but I'm not exactly sure what to change it to since there aren't really any other global variables in the IB. Maybe IB.colour_mode? Or maybe colour_mode should user getters and setters?